src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/data/VisitDto.kt

```diff
@@ -23,6 +23,10 @@ data class VisitDto(
   val visitStatus: String,
   @Schema(description = "Visit Status Description", example = "Reserved", required = true)
   val visitStatusDescription: String,
+  @Schema(description = "Visit Restriction", example = "OPEN", required = true)
+  val visitRestriction: String,
+  @Schema(description = "Visit Restriction Description", example = "Open", required = true)
+  val visitRestrictionDescription: String,
```

Comment:
arr now I see why it is not a enum,... but I do wonder why we are passing back a description,.. isn't this a ui concern

Reply:
agree - i wonder if the UI team originally wanted the backend to provide the localised strings.  I have suggested externalised content management if this is the case.